src/api/users/userLogin.js

```diff
@@ -17,11 +17,13 @@ export const userLogin = (mail, password) => {
   return fetch("https://dev.gyvit.io/api/user/token", request)
     .then(res => res.json())
     .then(response => {
+      console.log(response)
       //stock token in localStorage storeToken()
-      if (response.success === true) {
+      if (response.success) {
         storeToken(response.data.token)
+      } else {
+        storeToken("")
```

Comment:
no you should call a removeToken method that will correctly remove the token from localstorage